### DISCUSSION TITLE
Stuck At 5fps

### DISCUSSION THREAD

**Discussion by Kingmilton7** [opentrack/opentrack#1849] [Q&A]
Im new to this and im trying to play flight sims but its so laggy i dont know to increase the fps pls help
![Screenshot 2024-05-15 191757](https://github.com/opentrack/opentrack/assets/48865333/d71decaf-55a0-4acd-a2fe-d2082af9fad3)

**Comment by DaWelter** [comment]
The resolution. You gain nearly nothing by having it this high. Decrease it to 640 x 480 or even lower.
The actual pose network actually looks only at a crop rescaled to 129x129!

**Comment by kakra** [comment]
I'm only seeing 15 fps with 1ms processing time, the camera can do 30 fps, and there's still plenty of processing time left per frame. Increasing the threads doesn't help at all. Even with just one thread, it's using 3-4ms per frame.
![image](https://github.com/user-attachments/assets/99ff7e46-3932-4a96-b153-5e01b95a3345)
(resolution is 640x480)

**Reply by sthalik** [reply]
Exposure too high.

**Reply by kakra** [reply]
Hmm, okay but I cannot change it: "camera settings" does not open. I'll check what I could do about it...

**Reply by sthalik** [reply]
Is this Linux? Then use `qv4l2`.

**Reply by kakra** [reply]
Yes, Linux. It's in v2l-utils for me. But I need to reboot first to test it, I just upgraded the nvidia driver, and now all sorts of graphic acceleration won't work. I'll come back here later. Thanks for pointing me to the tool. I never used a webcam, just bought it for opentrack a few days ago. ;-)

**Reply by kakra** [reply]
Okay, it's not exposure itself, it's the exposure dynamic frame rate - switched it off, 30 fps:
![image](https://github.com/user-attachments/assets/633010c9-ceef-40e7-b652-452b4c9d482e)
